packages/compiler/lib/decorators.ts

```diff
@@ -961,7 +961,7 @@ export function getOverloadedOperation(
   return program.stateMap(overloadsOperationKey).get(operation);
 }
 
-const projectedNameKey = Symbol("projectedNameKey");
+const projectedNameKey = createStateSymbol("projectedNameKey");
```

Comment:
I remember @timotheeguerin saying that `Symbol("...")` does result in subtle bugs and that we should stick with `createStateSymbol`. There seem to be a lot of other instances of raw Symbol being used, most in `cadl-azure` but also in projections and a couple other places. Should we update them all?

Reply:
I looked at the others in core and they are OK. But I will look at cadl-azure.

tmlanguage-generator: completely separate tool
projection-logic.test: test only 
effective-type.test: test only
checker: for union variant name: deliberately a unique, non-shared symbol

But let me know if you see others in core that my search missed.

Reply:
Nope those were the core ones I saw. Thanks for looking 👍 

The cadl-azure ones do look to be all throughout the libraries in non-test code.

Reply:
I'm happy to make a RP

Reply:
Yes, we should fix those in cadl-azure and thank you for offering to make the PR. I'll take you up on that. :)

Reply:
Here's that PR: 
https://github.com/Azure/cadl-azure/pull/2556